server/server.go

```diff
@@ -475,7 +475,10 @@ func (s *Server) appendSensuService() {
 func (s *Server) appendSlackService() {
 	c := s.config.Slack
 	l := s.LogService.NewLogger("[slack] ", log.LstdFlags)
-	srv := slack.NewService(c, l)
+	srv, err := slack.NewService(c, l)
+	if err != nil {
+		s.Logger.Println("W! Slack init :", err)
```

Comment:
Go ahead and return the error here and check it in NewServer and return it from there. That will cause the server to print the error message and shutdown, which is the behavior we want.